backend/src/test/java/com/bakdata/conquery/api/StoredQueriesProcessorTest.java

```diff
@@ -175,6 +177,11 @@ private static ManagedQuery mockManagedQuery(IQuery queryDescription, User user,
 	private static ExecutionStatus makeState(ManagedExecutionId id, User owner, User callingUser, ExecutionState state, String typeLabel, SecondaryIdDescriptionId secondaryId) {
 		OverviewExecutionStatus status = new OverviewExecutionStatus();
 
+		final ManagedQuery execMock = new ManagedQuery() {{

```

Comment:
vorsicht, so zu instanziieren macht aus irgendeinem grund instanceof checks bananas

Reply:
Komisch, eigentlich sollte das eine Anonyme Klasse erzeugen, aber Instance of sollte dennoch gehen. TypeIdResolving sollte aber nicht mehr gehen

Reply:
vlt war es auch typeidresolving, hatte auf jeden fall probleme